src/pages/integrations/wallet-connect/wallet-connect.html

```diff
@@ -0,0 +1,186 @@
+    </ion-list>
+
+    <button *ngIf="!peerMeta || !peerMeta.name || !uri" ion-button class="button-standard" (click)="initWalletConnect()">
+      {{'Connect' | translate}}
+    </button>
+  </div>
+
+  <div *ngIf="connected">
+
+    <ion-list class="bp-list">
+      <ion-item *ngIf="peerMeta && peerMeta.name" class="sub-title">
+        <ion-label>
+          <div class="main-label">
+            <span class="peer-meta-name">{{ peerMeta.name | uppercase }} </span>
+          </div>
+        </ion-label>
+        <div class="last-item" item-end>
+          {{ getChainData(activeChainId).name }}
+        </div>
+      </ion-item>
+
+      <ion-item *ngIf="peerMeta && peerMeta.name" class="sub-title">
+        <ion-label>
+          <div class="main-label">
+            {{ 'Connected with' | translate}}
+          </div>
+        </ion-label>
+        <ion-note item-end>
+          <ion-row align-items-center class="wallet" *ngIf="wallet">
+            <ion-col>
+              <coin-icon [coin]="wallet.coin" [network]="wallet.network"></coin-icon>
+            </ion-col>
+
+            <ion-col>
+              <span class="note-container ellipsis">{{wallet.name}}</span>
+            </ion-col>
+          </ion-row>
+        </ion-note>
+      </ion-item>
+
+      <ion-item class="sub-title">
+        <ion-label>
+          <div class="main-label">{{ 'Pending Call Requests' | translate }}</div>
+        </ion-label>
+      </ion-item>
+
+      <action-card *ngIf="requests.length === 0" class='zero-state'>
+        <div action-card-title class="main-label">
+          {{'No pending call requests' | translate}}
+        </div>
+      </action-card>
+
+      <div *ngFor="let request of requests">
+        <action-card>
+          <div action-card-title class="main-label">
+            <div>
+              {{ 'Id' }}
+              <ion-note item-end>
+                {{ request.id }}
+              </ion-note>
+            </div>
+            <div (click)="rejectRequest(request)">
+              <ion-icon ios="md-close" md="md-close"></ion-icon>
+            </div>
+          </div>
+          <div action-card-body>
+            <div class="item" *ngIf="request.method">{{ 'Method' | translate }}</div>
+            <ion-note *ngIf="request.method" item-end>
+              {{ request.method }}
+            </ion-note>
+            <div *ngFor="let param of request.params; let i = index;">
+              <div *ngIf="request.method === 'eth_signTransaction' || request.method === 'eth_sendTransaction'">
+                <div class="item" *ngIf="param.to">
+                  <div>{{ 'To' | translate }}</div>
+                  <ion-note class="ellipsis" item-end>
+                    {{ param.to }}
+                  </ion-note>
+                </div>
+                <div class="item" *ngIf="param.gas">
+                  <div>{{ 'Gas Limit' | translate }}</div>
+                  <ion-note item-end>
+                    {{ param.gas }}
```

Comment:
Should we use `satToUnit` pipe with Gas, Gas limit or Gas Price?

Reply:
no with Gas Limit